File: app/src/main/java/com/sunny/family/home/MainActivity.java
package com.sunny.family.home;

import android.os.Bundle;

import com.sunny.family.R;
import com.sunny.family.livedata.LiveDataFragment;
import com.sunny.lib.base.BaseActivity;

import org.jetbrains.annotations.Nullable;

/**
 * Created by zhangxin17 on 2020/7/13
 */
public class MainActivity extends BaseActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.act_main);
        getSupportFragmentManager().beginTransaction()
                .add(R.id.act_main, new LiveDataFragment())
                .commit();

    }
}

File: moduleNfc/src/main/java/com/sunny/module/nfc/MainActivity.kt
package com.sunny.module.nfc

import android.content.Intent
import android.os.Bundle
import com.sunny.lib.common.base.BaseActivity
import com.sunny.module.nfc.service.NfcServiceActivity
import kotlinx.android.synthetic.main.activity_main.*

/**
 * https://developer.android.com/guide/topics/connectivity/nfc?hl=zh-cn
 */
class MainActivity : BaseActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        initView()
    }

    private fun initView() {
        btn_client.setOnClickListener {
            startActivity(Intent(mmActivity, NfcClientActivity::class.java))
        }

        btn_service.setOnClickListener {
            startActivity(Intent(mmActivity, NfcServiceActivity::class.java))
        }

        btn_all.setOnClickListener {
            startActivity(Intent(mmActivity, NfcTechActivity::class.java))
        }
    }
}